unit-tests/py/rspy/devices.py

```diff
@@ -48,7 +48,16 @@ def __init__( self, sn, dev ):
         self._product_line = None
         if dev.supports( rs.camera_info.product_line ):
             self._product_line = dev.get_info( rs.camera_info.product_line )
-        self._port = acroname and _get_port_by_dev( dev )
+        self._physical_port = dev.supports( rs.camera_info.physical_port ) and dev.get_info( rs.camera_info.physical_port ) or None
```

Comment:
Maybe add parentheses so it's clear that the 'and' is before the 'or'?

Reply:
But `and` is always of greater precedence than `or` :)